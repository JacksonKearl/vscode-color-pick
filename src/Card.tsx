import * as React from "react";

interface ICardProps {
  children: {};
}

export default class Card extends React.Component {
  public props: ICardProps;

  constructor(props: ICardProps) {
    super(props);
    this.props = props;
  }

  public render() {
    return (
    <div
      style={{
        alignItems: "flex-start",
        color: "black",
        background: "white",
        borderRadius: "4px",
        boxShadow:
          "rgba(0, 0, 0, 0.15) 0px 0px 0px 1px, rgba(0, 0, 0, 0.15) 0px 8px 16px",
        boxSizing: "initial",
        display: "flex",
        flexDirection: "column",
        justifyContent: "center",
        marginTop: "16px",
        padding: "10px",
        width: "200px",
        fontSize: "16px"
      }}
    >
      {this.props.children}
    </div>);
  }
}
