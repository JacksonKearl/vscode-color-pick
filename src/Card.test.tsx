import * as React from 'react';
import { create } from 'react-test-renderer';
import Card from './Card';


describe('Card component', () => {
  it('matches snapshot', () => {
    const Component = create(<Card><p>Card</p></Card>);
    expect(Component.toJSON()).toMatchSnapshot();
  });
});
