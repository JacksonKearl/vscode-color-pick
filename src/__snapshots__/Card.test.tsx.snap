// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component matches snapshot 1`] = `
<div
  style={
    Object {
      "alignItems": "flex-start",
      "background": "white",
      "borderRadius": "4px",
      "boxShadow": "rgba(0, 0, 0, 0.15) 0px 0px 0px 1px, rgba(0, 0, 0, 0.15) 0px 8px 16px",
      "boxSizing": "initial",
      "color": "black",
      "display": "flex",
      "flexDirection": "column",
      "fontSize": "16px",
      "justifyContent": "center",
      "marginTop": "16px",
      "padding": "10px",
      "width": "200px",
    }
  }
>
  <p>
    Card
  </p>
</div>
`;
